lennoxs30api/s30api_async.py

```diff
@@ -469,6 +469,7 @@ async def messagePump(self) -> None:
         # Observations:  the clientId is not passed in, they must be mapping the token to the clientId as part of negotiate
         # TODO: The long polling is not working, I have tried adjusting the long polling delay.  Long polling seems to work from the IOS App, not sure
         # what the difference is.   https://gist.github.com/rcarmo/3f0772f2cbe0612b699dcbb839edabeb
+        # Returns True if messages were received, False if no messages were found, and throws S30Exception for errors
```

Comment:
Thanks for adding the comment - I like this.  

A difference in behavior is on local when no messages are available a 204 is sent.  From the cloud we get a 200 with an empty message response - like this:

{
    "messages": []
}

We'd need a check before the for loop in the 200 response to look for zero messages and return false?

Reply:
Good point, I added this check.